extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -146,6 +151,20 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
                                   'data-amp-slot-index')));
     return reporter;
   }
+
+  /**
+   * @param {string} size
+   * @return {string} The ad unit hash key string.
+   * @private
+   */
+  adKey_(size) {
+    const element = this.element;
+    const domFingerprint = domFingerprintString(element);
```

Comment:
Again, the whole thing is going to be hashed to get the adk. I preferred hashing the concatenated string, rather than hashing some stuff concatenated with a hash.